R/metadata.R

```diff
@@ -46,10 +54,17 @@ setClass(
     load.rate.units="character",
     # Other
     station="character",
+    wq.sta.id="character",
+    station.lat="ANY",
+    station.lon="ANY",
+    disch.sta.id="character",
+    disch.basin.area="ANY",
+    wq.basin.area="ANY",
```

Comment:
and actually, could you add an additional field for `basin.area.units`? then we can take care of more units conversions for the user if needed, and can report yield in whichever areal units they prefer